R/lazyframe-frame.R

```diff
@@ -332,20 +662,98 @@ lazyframe__sort <- function(
 #'   })
 #' }
 lazyframe__with_columns <- function(...) {
+  structify <- parse_env_auto_structify()
+
+  parse_into_list_of_expressions(..., `__structify` = structify) |>
+    self$`_ldf`$with_columns() |>
+    wrap()
+}
```

Comment:
Why

Reply:
I didn't change this code, it's just git showing diffs in a weird way (the code is almost the same for `with_columns_seq()`, hence the confusion I guess)